mozilla-release/browser/components/customizableui/CustomizableUI.jsm

```diff
@@ -307,20 +196,10 @@ var CustomizableUIInternal = {
       "home-button",
       "spring",
       "urlbar-container",
-<<<<<<< HEAD
-#if 0
-      "search-container",
-#endif
-      "cliqz-vd-btn", // Cliqz video downloader 
+      "spring",
+      "cliqz-vd-btn", // Cliqz video downloader
```

Comment:
in extension 1.22, video-downloader is managed like page action and is not a subject for customizable UI. 

why this code is here in the first place?

Reply:
Have no idea, it is always was in this way. @mai-cliqz can you comment this?

Reply:
After the fix that I made on the extension, this code is no longer needed. We can remove this "cliqz-vd-btn" here